### PR TITLE
Melinda Sample

### PR DESCRIPTION
**Pull request by beefaloe**
Write comments to the instructor here

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # wdd130-lab-1.1
 
-test
```
Comment (jstaso) [inline README.md:3]: Remove this

File: index.html [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<h1>Hello World!</h1>
```
Comment (jstaso) [inline index.html]: Remove this

File: README.md (hunk 1 above)
Comment (beefaloe) [reply README.md:3]: Removed

File: index.html (hunk 2 above)
Comment (beefaloe) [reply index.html]: Removed

**Review by jstaso — APPROVED** [review]
You are free to merge!

**Pull request merged**